src/components/Text/utils.ts

```diff
@@ -22,3 +22,7 @@ export const determineDefaultTextElement = (variant: TextVariants = TextVariants
 
   return 'span';
 };
+
+export const determineTextAlignment = (alignment: TextAlignments = TextAlignments.center): string => {
```

Comment:
do we need to abstract this? Since we are passing a default to the align prop and this is specifically a text util may be easier to just put the return value as the class?